src/inc/clrconfigvalues.h

```diff
@@ -650,7 +650,8 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_HillClimbing_GainExponent,
 /// Tiered Compilation
 ///
 #ifdef FEATURE_TIERED_COMPILATION
-RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_TieredCompilation, W("EXPERIMENTAL_TieredCompilation"), 0, "Enables tiered compilation")
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_TieredCompilation, W("TieredCompilation"), 0, "Enables tiered compilation")
+RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_LEGACY_TieredCompilation, W("EXPERIMENTAL_TieredCompilation"), 0, "Deprecated - Use COMPLUS_TieredCompilation")
```

Comment:
It looks like netci.groovy at the root also has two references to the old name

Reply:
Was wondering why you would miss that, apparently that is not checked in :)

Reply:
Oh nevermind, just missed it: https://github.com/dotnet/coreclr/blob/master/netci.groovy#L72-L102

Reply:
Sorry for the confusion, it looks like netci.groovy still has a reference to the old name and is missing from the change. I don't know what it is used for, should it be updated?

Reply:
netci.groovy is used to specify the Jenkins CI jobs (https://ci.dot.net/job/dotnet_coreclr/job/master/); it should be updated.